src/Symfony/Component/ErrorHandler/DebugClassLoader.php

```diff
@@ -606,6 +615,10 @@ public function checkAnnotations(\ReflectionClass $refl, string $class): array
                 }
             }
 
+            if (isset($doc['return-typehint-will-change'])) {
```

Comment:
this should say `type`, not `typehint`, to match the way it is named in PHP (PHP does not have a concept of typehint, but of type)

Reply:
fixed